php-files/functions.php

```diff
@@ -56,7 +56,41 @@ function displayFish(array $fishFromDB):string
                 . $fish['pattern']
                 . '</h3></div>';
         } else {
-            return 'Error! Please contact administrator';}
+            return 'Error! Please contact administrator';
+        }
     }
     return $result;
 }
+
+/**
+ * @param $dataBase
+ * @param array $fishArray
+ * @return bool
+ * Takes an array and inserts the data into the database
+ */
+function insertEntryIntoDB($dataBase, array $fishArray): bool
+{
```

Comment:
Fails silently, input should be validated before the call to insertEntryIntoDB happens in newFish.php